### DISCUSSION TITLE
po-table: linha selecionada sem radio ou checkbox

### DISCUSSION THREAD

**Discussion by william-sade** [po-ui/po-angular#1540] [Angular] [answered]
É possível selecionar uma linha do po-table sem a necessidade de ter uma 'coluna' radio/check para isso?

Motivo:
É importante que o usuário saiba qual registro ele está analisando.
Setar as propriedades **p-selectable** e **p-single-select** para true cria um radio totalmente desnecessário para a nossa visão.

Obrigado.

**Comment by william-sade** [comment]
Alguma posição sobre esta dúvida?

**Comment by anliben** [comment, marked as answer]
Boa Tarde @william-sade, tudo bem?

Atualmente nossa tabela não remove o rádio na linha. Para ver todas as possibilidades podes ir à documentação na aba de samples

https://po-ui.io/documentation/po-table

Podes também remover utilizando css.

Attr.

**Comment by william-sade** [comment]
Boa tarde.
Obrigado pelo retorno.
Esta então seria a melhor opção?

`.po-table-column-selectable {
  display: none;
}`

**Reply by anliben** [reply]
Boa Tarde @william-sade, tudo bem?

Isso mesmo, pode ser retirado desta forma.
